contracts/governance/gov_constants.cairo

```diff
@@ -0,0 +1,10 @@
+const PROPOSAL_VOTING_TIME_BLOCKS = 50; // TODO what is the blocktime on mainnet precisely? 30 mins? any commitment from Starkware? // TODO make higher after testing
```

Comment:
shouldn't be higher if we're going live soon?

Reply:
making it 500 blocks now, around 26 hours with current mainnet speed.